EIPS/eip-erc20-swiss-compl.md

```diff
@@ -0,0 +1,182 @@
+  * @return true if the address has been removed from issuers,
+  * false if the address wasn't in the issuer list in the first place
+  */
+  function removeIssuer(address _operator) external;
+
+  /**
+  * @dev Allows the current issuer to transfer his role to a newIssuer.
+  * this operation can be performmed only by an Issuer.
+  * @param _newIssuer The address to transfer the issuer role to.
+  */
+  function transferIssuer(address _newIssuer) external;
+
+}
+```
+
+The ERCtoBeAssigned extends ERC-20. Due to the indivisible nature of asset tokens, the decimals number MUST be zero.
+
+### Whitelist and Frozenlist
+
+The accomplishment of the Swiss Law requirements is achieved by the use of two distinct lists of address: the Whitelist and the Frozenlist.
+Although these lists may look similar, they differ for the following reasons: the Whitelist members are the only ones who can receive tokens from other addresses. There is no restriction on the possibility that these addresses can transfer the tokens already in their ownership.
+On the other hand, the addresses assigned to the Frozenlist have to be considered "frozen", so they cannot either receive tokens or send tokens to anyone.
+
+### Issuers
+
+A key role is played by the Issuer (if there are more than one, we refer to them as Issuers). This figure has the permission to manage Whitelists and Frozenlists, to revoke tokens and reassign them and to transfer the role to another address. Issuers are nominated by the Owner of the contract, who also is in charge of remove the role. By default, the Owner SHOULD NOT be an Issuer, but in some cases the Owner can manage both roles without restrictions.
+
+### Revoke and Reassign
+
+Revoke and Reassign methods allow Issuers to move tokens from addresses, even if they are in the Frozenlist. The Revoke method transfers the entire balance of the target address to the Issuer who invoked the method. The Reassign method transfers the entire balance of the target address to another address. These rights of these operations MUST be allowed only to Issuers.
+
+## Rationale
+
+There are currently no token standards that expressly facilitate conformity to securities law and related regulations. EIP-1404 (Simple Restricted Token Standard) it’s not enough to address FINMA requirements around re-issuing securities to Investors.
+In Swiss law, an issuer must eventually enforce the restrictions of their token transfer with a “freeze” function. The token must be “revocable”, and we need to apply a white-list method for AML/KYC checks.
+
+## Backwards Compatibility
+
+This EIP does not introduce backward incompatibilities and is backward compatible with the older ERC-20 token standard.
+This standard allows the implementation of ERC-20 functions transfer, transferFrom, approve and allowance alongside to make a token fully compatible with ERC-20.
+The token MAY implement decimals() for backward compatibility with ERC-20. If implemented, it MUST always return 0.
```

Comment:
Why must decimals be `0`?  If this is a hard requirement, it should probably be mentioned in the specification.